<launch>

  <include file="$(find turtlebot_bringup)/launch/minimal.launch" />
  <include file="$(find rplidar_ros)/launch/rplidar.launch" />
  <node pkg="tf" type="static_transform_publisher" name="base_link_to_laser" args="0 0 0 0 0 0 base_link laser 100" />
  <include file="$(find turtlebot_teleop)/launch/logitech.launch" />

  <arg name="geotiff_map_file_path" default="$(find hector_geotiff)/maps"/>
  <include file="$(find hector_mapping)/launch/mapping_default.launch"/>
  <include file="$(find hector_geotiff)/launch/geotiff_mapper.launch">
    <arg name="trajectory_source_frame_name" value="scanmatcher_frame"/>
    <arg name="map_file_path" value="$(arg geotiff_map_file_path)"/>
  </include>
  <node pkg="rviz" type="rviz" name="rviz" 
    args="-d $(find hector_slam_launch)/rviz_cfg/mapping_demo.rviz"/>
</launch>
